site/tsconfig.json

```diff
@@ -16,7 +16,10 @@
 		"skipLibCheck": true,
 		"strict": true,
 		"target": "es2020",
-		"baseUrl": "src/"
+		"baseUrl": "src/",
+		"paths": {
+			"@/*": ["./*"]
+		}
```

Comment:
🙅‍♀️

Reply:
I can remove this, but still need to figure out the best way to import shadcn components with using the @ in the paths